modules/prometheus/src/main/scala/com/evolutiongaming/smetrics/CollectorRegistryPrometheus.scala

```diff
@@ -4,11 +4,33 @@ import cats.effect.{Resource, Sync}
 import cats.implicits._
 import io.prometheus.client.Collector
 import io.prometheus.{client => P}
+import com.evolutiongaming.smetrics.CollectionHelper._
 
 object CollectorRegistryPrometheus {
 
   def apply[F[_] : Sync](collectorRegistry: P.CollectorRegistry): CollectorRegistry[F] = {
 
+    def initializeLabelValues[Child, C <: P.SimpleCollector[Child]](
+      collector: C,
+      initialLabelValues: List[List[String]]
+    ): Resource[F, Unit] =
+      if (initialLabelValues.nonEmpty)
+        Resource.liftF(
+          Sync[F].delay {
+            initialLabelValues
+              .combinations()
```

Comment:
1. `combinations()` -> `combinations` as it's a pure method
2. because the method is a pure - no need to wrap into `Sync[F].delay { }` 

```scala
val combinations = initialLabelValues.combinations()
Sync[F]
  .delay { combinations.foreach(labelValues => collector.labels(labelValues: _*)) }
  .toResource
}
```

Reply:
Unfortunately, I couldn't find any information about '.toResource'.

Done

Reply:
> Unfortunately, I couldn't find any information about '.toResource'.

```scala
import com.evolutiongaming.catshelper.CatsHelper._
```